parity/deprecated.rs

```diff
@@ -203,6 +203,12 @@ pub fn find_deprecated(args: &Args) -> Vec<Deprecated> {
 		result.push(Deprecated::Removed("--tx-queue-ban-time"));
 	}
 
+	// Removed in 2.0.
+
+	if args.flag_fast_and_loose {
+		result.push(Deprecated::DoesNothing("--fast-and-loose"));
```

Comment:
Minor nit, but why `DoesNothing` instead of `Removed`?

Reply:
Correct. It should be removed.